src/java.base/share/classes/sun/security/ssl/SSLSocketImpl.java

```diff
@@ -587,7 +587,7 @@ public void close() throws IOException {
         } catch (IOException ioe) {
             // ignore the exception
             if (SSLLogger.isOn && SSLLogger.isOn("ssl")) {
-                SSLLogger.warning("SSLSocket duplex close failed", ioe);
+                SSLLogger.warning("SSLSocket duplex close failed. Debug info only. Exception details:", ioe);
```

Comment:
If this is a debug message, shouldn't we just use `SSLLogger.fine()` instead of `SSLLogger.warning()`, with the same message "SSLSocket duplex close failed"? @coffeys what do you think?

Reply:
@seanjmullan  - It's the exception stacktrace printed after the message that's causing issue for some. Some are reading it as a JDK functionality issue (when it's not)

I've no strong preference whether this should be printed at warning() or fine() level. That granularity is largely broken since JDK 11 - people need to use -Djavax.net.debug=all to get any reasonable amount of data. It would be good to see https://bugs.openjdk.java.net/browse/JDK-8044609 worked on.

I think the patch looks ok as ie. Evan has fixed up some issues with the test - we should expect a new version shortly.

Reply:
Hi @seanjmullan @coffeys 

I also don't have a logging level preference, so if there's merit to changing it, I'll be happy to do so.
I've updated the test case and verified it passes on all platforms.

Looking forward to any further feedback :)

Reply:
Ok, It sounds like whether it should be a different logging level (fine) or not can be handled separately as part of a more global effort across other SSL logging messages. So I am ok with the change as-is.

Reply:
Thanks Sean!